cdap-ui/app/directives/complex-schema/complex-schema.js

```diff
@@ -178,10 +188,11 @@ angular.module(PKG.name+'.commons')
       model: '=ngModel',
       recordName: '=',
       parentFormatOutput: '&',
-      isDisabled: '='
+      isDisabled: '=',
+      schemaPrefix: '='
     },
     link: (scope, element) => {
-      $compile('<my-complex-schema ng-model="model" record-name="recordName" hide-header="true" parent-format-output="parentFormatOutput()" is-disabled="isDisabled"></my-complex-schema')(scope, (cloned) => {
+      $compile('<my-complex-schema ng-model="model" record-name="recordName" hide-header="true" parent-format-output="parentFormatOutput()" is-disabled="isDisabled" schema-prefix="schemaPrefix"></my-complex-schema')(scope, (cloned) => {
```

Comment:
Can we try es6 string literal? Then we could have all the formatting. Right now it just expands as one single long line in the editor.